packages/core/src/schema/models/evaluationResults.ts

```diff
@@ -17,6 +17,7 @@ export const evaluationResults = latitudeSchema.table(
   'evaluation_results',
   {
     id: bigserial('id', { mode: 'number' }).notNull().primaryKey(),
+    uuid: uuid('uuid').unique(),
```

Comment:
For now until we implement proper uuid pasing from `chainRun` the uuid is auto generated when the evaluation result is created.